### PR TITLE
chore: make the changelog format less noisy

### LINKED ISSUE #416: Improve changeset default template
### Motivation

In changelog, we should show most relevant information first and make it cleaner to read

- Description of change should go first
- Remove commit link, as we already have PR link
- Remove the text `All notable changes to this project will be documented in this file.
See Conventional Commits for commit guidelines.` that repeats itself a lot of times (for example [here](https://github.com/FuelLabs/fuels-ts/pull/415))

### Possible implementations

For example:
Change from:
https://github.com/FuelLabs/fuels-ts/pull/405 [19dee43](https://github.com/FuelLabs/fuels-ts/commit/19dee437f0ea2fe02a4a7f56b0b55d84279c2dc0) Thanks [@camsjams](https://github.com/camsjams)! - Bumping all packages to next minor version

to:
Bumping all packages to next minor version - Thanks [@camsjams](https://github.com/camsjams)! - https://github.com/FuelLabs/fuels-ts/pull/405

### PR DESCRIPTION
**Pull request by Dhaiwat10**
## Summary
This PR tweaks our changelog's format, mainly omitting unimportant changes like dependency bumps from our changelogs to make them way less noisy. Nothing fancy, I have just re-used the config used by the [fuel-ui repo](https://github.com/FuelLabs/fuel-ui/blob/master/scripts/changesets/changelog.js).

tldr; we won't see any more of this:
image

Wanted to get this out of the way before I work on a solution for #970.

Closes #416

### CODE REVIEW

**Comment by danielbate** [comment]
Can this close #416 ?

**Review by danielbate — DISMISSED** [review]

**Review by arboleya — DISMISSED** [review]
Amazing! 🙏

**Review by arboleya — APPROVED** [review]

**Review by Torres-ssf — APPROVED** [review]
